lbrynet/daemon/Daemon.py

```diff
@@ -2430,13 +2431,7 @@ def jsonrpc_wallet_send(self, amount, address=None, claim_id=None):
             raise Exception("Given both an address and a claim id")
         elif not address and not claim_id:
             raise Exception("Not given an address or a claim id")
-
-        try:
-            amount = Decimal(str(amount))
-        except InvalidOperation:
-            raise TypeError("Amount does not represent a valid decimal.")
-
-        if amount < 0:
+        if amount < 0.0:
             raise NegativeFundsError()
         elif not amount:
             raise NullFundsError()
```

Comment:
Same as earlier comment, this is doing less than comparison before a null comparison. Also, why the change from `0` to `0.0`? The amount should be in dewies which is an `int`.